project_silentdata.md

```diff
@@ -0,0 +1,188 @@
+﻿# Silent Data Polkadot Integration 
+ 
+ **Team Name**: Applied Blockchain Ltd
+
+**Payment Address**: USDC (Ethereum) address: 0x91a5ade2522ac8c3761922a4255e0fef89116a37
+
+**[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):**(https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 2 
+
+## Project Overview :page_facing_up:
+
+This application is in response to an RFP 
+
+### Overview
+
+Applied Blockchain has developed Silent Data as a platform for proving properties of private off-chain (web2) data in blockchain smart contract (web3) applications (dApps).
+
+Silent Data leverages hardware secure enclaves with attestation, in particular, Intel SGX in order to enable privacy-preserving retrieval and processing of off-chain data, and generation of cryptographic proofs that are verifiable in blockchain smart contracts. This ensures that sensitive information is never revealed, not even to those hosting the platform, and code attestation with a smart contract link ensures that the code used to retrieve the data and generate the proofs cannot be modified or interfered with by the operator.
+
+Silent Data proof certificates are powerful because they can verifiably demonstrate that private data meets certain requirements without the need to reveal that data, and without that data being accessible to Silent Data operators. The service is initially centrally hosted, and will be decentralised over time in order so that it cannot be censored by the operator.
+
+Silent Data enables verification of web2 account ownership, including social media services such as Instagram. This enables enrichment of web3 identities and assets with attestations from the web2 world, for example, if someone is using web3 to purchase from a brand with a web2 presence.
+
+NFT creators with verified Instagram accounts, for example, can use Silent Data to prove they are behind a specific wallet. This will help to reduce fraud in NFT and web3 commerce.
+
+Silent Data will also be extended to enable verification of web2 financial credentials, to augment web3 DeFi and tokenisation dapps.
+
+**An indication of how your project relates to / integrates into Substrate / Polkadot / Kusama.**
+
+Silent Data provides privacy-preserving cryptographic proofs about properties of web2 data. We refer to the output as a “proof certificate”, consisting of a set of key-value pairs signed by a manufacturer attested hardware secure enclave. The signatures can be verified using various cryptography primitives available on different blockchains, including Polkadot (Substrate) and Ethereum. A blockchain wallet is associated with a Silent Data proof certificate by having the wallet owner sign the private data that is encrypted and sent to the enclave with their wallet private key.
+
+Silent Data will integrate with a Substrate chain to provide maximum coverage within the Polkadot ecosystem.

```

Comment:
Can you explain what you mean by this? Are you planning to "partner" with an existing chain? "Maximum coverage" would probably be achieved if this project was compatible with more than just one chain.

Reply:
Hi Samuelle,
I confirm that by “maximise” we mean enabling Silent Data for all parachains.